### PR TITLE
changed ResultsTable to create a new table

### PR DESCRIPTION
**Pull request by cwood1967**
We were writing a macro to loop over a list of file and output data to the main ResultsTable. StackReg was called for each file, and the call to TurboReg from StackReg caused the table to be reset. I changed TurboReg to create a new instance of ResultsTable rather than getting the ResultsTable from Analyzer.

### CODE REVIEW

**Comment by ctrueden** [comment]
Sorry for the long delay in reviewing this. Your commit conflates a few unrelated changes, including changes to the `pom.xml` as well as some source comments. Take a look at the diff to see what I mean. Would you please [rewrite it to be clean](http://imagej.net/Git#Rewriting_history)?

File: src/main/java/TurboReg_.java [hunk 1]
```diff
@@ -259,7 +262,7 @@ public ImagePlus getTransformedImage (
  <td></td><td>targetPoints[0][1]</td><td>17</td><td>floating-point (TRSAB)</td>
  </tr><tr>
  <td></td><td>sourcePoints[1][0]</td><td>18</td><td>floating-point (RSAB)</td>
- </tr><tr>
+ </tr><tr>clear
```
Comment (ctrueden) [inline src/main/java/TurboReg_.java]: This looks like a typo.

File: src/main/java/TurboReg_.java [hunk 2]
```diff
@@ -156,6 +156,7 @@
  (3), and bilinear transformation (4).
  @see TurboReg_#run
  @see TurboReg_#getTargetPoints
+ @return Source points double array
```
Comment (ctrueden) [inline src/main/java/TurboReg_.java:159]: These javadoc improvements should be a separate commit.

File: pom.xml [hunk 3]
```diff
@@ -20,6 +20,29 @@
 	<url>http://fiji.sc/TurboReg</url>
 	<inceptionYear>2010</inceptionYear>
 
+	<!--
```
Comment (ctrueden) [inline pom.xml:23]: This javadoc configuration fix should be its own commit, before the others.

File: .gitignore [hunk 4]
```diff
@@ -5,3 +5,7 @@
 
 # Maven #
 /target/
+
+# IntelliJ
```
Comment (ctrueden) [inline .gitignore:9]: These ignore rules should be a separate commit.

**Comment by ctrueden** [comment]
Note that the email address you used to do that commit is not associated with your GitHub account, so when you browse the commit it does not link back to @cwood1967 like it should.

**Comment by cwood1967** [comment]
Thanks for the comments. I cleaned to the git commits as you suggested/

**Comment by ctrueden** [comment]
Thanks @cwood1967, much nicer. There was still something funny in the history—the old commit was still there, but then subsequent commits also had the same changes split up nicely—so I took the liberty of rebasing it and then merged the rebased version.

I will try to cut a new release of TurboReg later this week and upload it to Fiji.

Note that there are some licensing issues with this—the Fiji team are not the copyright holders of this plugin, and our [BIG license](http://imagej.net/BIG_License) only permits us to redistribute, not modify. So there is some question of whether these changes are kosher. You might want to email [Philippe Thévenaz](http://bigwww.epfl.ch/thevenaz/), the original author of TurboReg, letting him know about these fixes.

See also https://github.com/fiji/TurboReg2
